oarepo_communities/ui/communities_components/semantic-ui/less/oarepo_communities/themes/default/views/card.variables

```diff
@@ -0,0 +1,2 @@
+@largeDesktopScreenCardHeight: 21.76rem;
```

Comment:
this should probably be pixel units, images usually dont scale relative to font size

Reply:
These are some communities styles that were in RDM, so I thought I would place them in oarepo-communities in our case. It was a copy paste. Converting to pixels.